sufsara_tutorial.Rmd

```diff
@@ -26,86 +26,57 @@ Navigate to your [profile page](https://portal.live.surfresearchcloud.nl/profile
 
-![X2Go settings](figs/x2gosettings.PNG)
-
-In addition, in the *Connections* tab you can set the compression method to "4k-png". This is a lower-colour setting that saves bandwith and thus makes the desktop more responsive.
-
-Press OK, and from here on you can press the bubble on the right-hand side to launch the dektop of the virtual machine. You will see a *Host key verification failed* warning; this is expected every time you launch a new VM, answer *Yes*. You might also see a warning about shared folders and printers; this is safe to ignore. Turn off folder and printer sharing for the message to go away.
-
-![VM running through X2go](figs/x2gorunning.PNG)
-
-<!-- Likely all deprecated, and possibly also not relevant
-
-## Shut down a VM
-
-**Very important: Remember to properly shut down VMs that you are not actively using, as they take up precious resources from the cloud! If you do not, other students might not be able to start their own VMs.**
+# Optional: accessing the VM from your own machine
 
-To properly shut down your VM, first you need to make sure your computer is no longer connected to it. If you have X2Go running, open the X2Go Client window again, and press the *Terminate* button on the lower right part of the main pane. This will gracefully log your user out.
+The tutorial above allows you to access the VM in-browser. There are, however, alternatives to access the VM directly from your own machine, for which we provide a tutorial below.
 
-Then on the SURFSara website, select your VM, press the *Power off* button and confirm *Send the power off signal*. Wait for the state to become *OFF*, then press the *Delete* button and confirm deletion.
+<!-- ## Enrol your SSH key -->
 
-**Important**: Only if your VM is **deleted** (does **not** show up in the *VMs* tab altogether) does it free the resources for other students to use!
+<!-- Navigate to your [SRAM profile page](https://sram.surf.nl/profile). Generate an SSH key on your own machine. Paste your *public* SSH key into the text box, and click the "Save" button. You should never paste the *private* key anywhere (it will also not be accepted if you try). -->
 
-## Saving changes
+<!-- Please refer to online tutorials to find/create an SSH key for your machine, like [this one](https://www.howtogeek.com/762863/how-to-generate-ssh-keys-in-windows-10-and-windows-11/) for Windows. -->
 
-If you change something, then restart the VM, you may notice that your changes have been lost. This is because by default VM hard drives are not persistent. You need to clone them and set them as persistent to make the changes stick.
+<!-- **Hint**: another easy way to find your SSH key is by running Git GUI on your own machine, through Help → Show SSH Key.  -->
```

Comment:
The reason why there are SSH keys in the first place is to connect using the command line (Bash). This is actually the most efficient way of connecting. This is because `ssh` allows forwarding ports to your own computer and the VMs have server software preinstalled. You can run `ssh -L8787:loclhost:8787 yourusername@yourserver.nl` to forward the port 8787 to your own computer, then in your own browser open http://localhost:8787 and immediately get to RStudio Server. This is very efficient, because then you are only sending commands over the internet, and not pictures, which is so much smaller. You also then don't have issues with repeated keystrokes etc. and you can even use RStudio to upload and download files. But the downside is that you need to run that one extra command and open a new tab in your browser.